front/pages/api/v1/w/[wId]/triggers/hooks/[webhookSourceId].ts

```diff
@@ -111,14 +117,64 @@ async function handler(
     });
   }
 
-  logger.info(
-    {
-      workspaceId: wId,
-      webhookSourceId,
+  // Fetch all triggers based on the webhook source id
+  const views = await WebhookSourcesViewResource.listByWebhookSource(
+    auth,
+    webhookSource.id
+  );
+
+  // Fetch all triggers based on the webhook source id
+  // flatten the triggers
+  const triggers = (
+    await concurrentExecutor(
+      views,
+      async (view) => {
+        const triggers = await TriggerResource.listByWebhookSourceViewId(
+          auth,
+          view.id
+        );
+        return triggers;
+      },
+      { concurrency: 10 }
+    )
+  ).flat();
+
+  const results = await concurrentExecutor(
+    triggers,
+    // Run every trigger.
+    async (trigger) => {
+      // Get the trigger's user and create a new authenticator
+      const user = await UserResource.fetchByModelId(trigger.editor);
+
+      if (!user) {
+        logger.error({ triggerId: trigger.id }, "Trigger editor not found.");
+        return new Err(new Error("Trigger editor not found."));
+      }
+
+      const auth = await Authenticator.fromUserIdAndWorkspaceId(user.sId, wId);
+      return launchAgentTriggerWorkflow({ auth, trigger });
     },
-    "[Webhook Trigger] Received webhook (skeleton handler)."
+    { concurrency: 10 }
   );
 
+  const errors = results.filter((result) => result.isErr());
+
+  if (errors.length > 0) {
+    logger.error({ errors }, "Error launching agent trigger workflows.");
+    return apiError(
+      req,
+      res,
+      {
+        status_code: 500,
+        api_error: {
+          type: "internal_server_error",
+          message: "Error launching agent trigger workflows.",
+        },
+      },
+      // Safe cast thanks to the .filter above.
+      (errors[0] as Err<Error>).error
+    );
+  }
```

Comment:
depending on the platform, returning an error might trigger a re-delivery of the webhook
this means we might trigger some agents multiple time if ONE of them failed for any reason

i think we want to return a success whatsoever ? tricky behaviour here we need to decide